docs/usage.md

```diff
@@ -291,6 +291,7 @@ nextflow run nf-core/nanoseq \
     --protocol cDNA \
     --skip_basecalling \
     --skip_demultiplexing \
+    --skip_qc \
```

Comment:
maybe --skip_alignment is better?

Reply:
I added both skip_qc (line 294) and skip_alignment (line 295) because the qc steps are for fastq files.